<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>571</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>571</width>
    <height>600</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string extracomment="Программное обеспечение конфертирования расписания занятий">schedule2html</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>favicon.ico</normaloff>favicon.ico</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>241</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Конвертирование Excel в html</string>
    </attribute>
    <widget class="QLineEdit" name="sourceExcelDirectoryPath">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>461</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="sourceDirectoryButton">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Выбрать</string>
     </property>
    </widget>
    <widget class="QPushButton" name="convertExcelToHTMLActionButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>170</y>
       <width>85</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Выполнить</string>
     </property>
    </widget>
    <widget class="QLabel" name="savingDirectoryLable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>381</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Выберете дирректорю сохранения расписаний в формате html</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="savingHTMLDirectoryPath">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>461</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="savingDirectoryButton">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>90</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Выбрать</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>451</width>
       <height>31</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="sourceDirectoryLable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>391</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Выберите дирректорию с расписанием файлов в формате excel</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="createNewDirectoryCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>181</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Создать новый подкаталог?</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Дополнительные функции</string>
    </attribute>
    <widget class="QLineEdit" name="JSONDirectoryPath">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="JSONDirectoryPathActionButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>60</y>
       <width>85</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Выполнить</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelJSONLavel_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>341</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Обновление JSON описания правил доступа к html файлам</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelJSONLavel_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Введите путь к каталогу с html файлами</string>
     </property>
    </widget>
    <widget class="QToolButton" name="JSONDirectoryPathChoiceButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="report">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>551</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="reportLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Отчет о выполнении</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
